### DISCUSSION TITLE
Actions workflows view should allow filtering by multiple statuses

### DISCUSSION THREAD

**Discussion by andrewring** [community/community#67516] [Actions]
### Select Topic Area

Product Feedback

### Body

Currently you can only filter by a single status, such as queued or in_progress. This makes it difficult to get an clear view of the work not yet complete without a lot of extra clicking. It should be possible to provide multiple values, matching any of which should be included in the results.

Fixing this should also enable this CLI feature request: https://github.com/cli/cli/issues/7949

**Comment by HarshDekate** [comment]
HEELO

**Comment by benmccown** [comment]
I'm kind of shocked this isn't a feature yet. When you select a status the UI shows you a checkbox, implying that you can select more than one status. There are literally 12 possible statuses a workflow run can be in- filtering on a single status is borderline useless.
